lib/jxl/encode.cc

```diff
@@ -1167,7 +1167,27 @@ JxlEncoderStatus JxlEncoderAddJPEGFrame(
     // Can't XYB encode a lossless JPEG.
     return JXL_ENC_ERROR;
   }
-
+  if (!io.blobs.exif.empty()) {
+    size_t exif_size = io.blobs.exif.size();
+    // Exif data in JPEG is limited to 64k
+    if (exif_size > 0xFFFF) return JXL_ENC_ERROR;
+    exif_size += 4;  // prefix 4 zero bytes for tiff offset
+    std::vector<uint8_t> exif(exif_size);
+    memcpy(exif.data() + 4, io.blobs.exif.data(), io.blobs.exif.size());
+    JxlEncoderUseBoxes(frame_settings->enc);
+    JxlEncoderAddBox(frame_settings->enc, "Exif", exif.data(), exif_size,
+                     JXL_TRUE);
```

Comment:
/*brotli=*/ or whatever the parameter name is :)

Reply:
updated.